src/index.html

```diff
@@ -11,7 +11,141 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <main>
+      <!-- <h1>HTML Form</h1> -->
+      <script type="text/javascript" src="./main.js"></script>
+      <form autocomplete="off" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+        <fieldset>
+          <legend>Personal information</legend> 
+          <div class="div-class">
+            <label>
+              Surname:
+              <input type="text"  name="surname" 
+                    minlength="2" maxlength="50"
+              >
```

Comment:
![image](https://user-images.githubusercontent.com/61904995/214073586-322c5042-4c15-40a5-ab53-c8c4325bcabc.png)
Check it all over page